docs/flow/reference/components/remote.s3.md

```diff
@@ -18,12 +29,14 @@ The following arguments are supported:
 
 Name | Type | Description                                                             | Default | Required
 ---- | ---- |-------------------------------------------------------------------------| ------- | --------
-`path` | `string` | Path in the format of `"s3://bucket/file"`                              | | **yes**
+`path` | `string` | Path in the format of `"s3://bucket/file"` | | **yes**
 `poll_frequency` | `duration` | How often to poll the file for changes, must be greater than 30 seconds | `"10m"` | no
-`is_secret` | `bool` | Marks the file as containing a [secret][]                               | `false` | no
-`client_options` | `client_options` | Additional [options](#client_options) for configuring the client      | | no
```

Comment:
Historically we haven't been documenting blocks as arguments. This might change, but I'd like to remove it to make it consistent with the other docs for now.

Reply:
How would we represent nested blocks? IE if client_options also had an `http_config`?

Reply:
Subheaders, maybe? That's a good point, though. Maybe we do something like this below the table: 

```
The following inner blocks can be specified:

* [`client_options`](#client_options): Additional options for configuring the client 
```

WDYT?

Reply:
Maybe a second table?

The issue IMO is that they _are_ different things: attributes are typed values, blocks are structural elements.

Reply:
I am down for a second table. I am unsure if they are different from a user's perspective. They are config. We can go with two tables and see if we get any feedback.

Reply:
We'll have to try to teach the users to see them as different, because one supports expressions and the other doesn't :)

Reply:
PTAL @mattdurham